### PR TITLE
Update dependency

### PR DESCRIPTION
**Pull request by eggplantzzz**
Update package to use the new org-scoped package `@truffle/contract` rather than `truffle-contract`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +30,7 @@
     "left-pad": "^1.1.3",
     "semver": "^5.3.0",
     "solidity-sha3": "^0.4.1",
-    "truffle-artifactor": "^2.1.2",
```
Comment (CruzMolina) [inline package.json:31]: Just checkin' in here: is the artifactor not used by the pkg?

Comment (eggplantzzz) [reply package.json:31]: If it is, I cannot see where it is used.

**Review by CruzMolina — APPROVED** [review]
🔥

**Comment by eggplantzzz** [comment]
It should be noted that I'm not really that sure how to test this. The test suite is in kind of an unknown state right now. @gnidan, do you know how we can test this?

**Pull request merged**